spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -112,11 +112,11 @@ protected List<Object[]> sql(String query, Object... args) {
     return rowsToJava(rows);
   }
 
-  protected List<Object[]> rowsToJava(List<Row> rows) {
-    return rows.stream().map(this::toJava).collect(Collectors.toList());
+  public static List<Object[]> rowsToJava(List<Row> rows) {
```

Comment:
Style: I feel its messy now to have this class now as inherited and a util class.  Especially some methods are changed to static and other methods are not, even though they call the static ones.

What do you think?  Maybe we can make a separate base class for the helper methods like 'SparkTestHelperBase' and have both SparkTestBase and your test inherit from it (to avoid changing all the tests)?

Reply:
Made the change per suggestion.